kitsune/src/service/post.rs

```diff
@@ -357,11 +457,261 @@ impl PostService {
             .await
             .map_err(Error::Event)?;
 
-        self.search_service.remove_from_index(post.into()).await?;
+        self.search_service.remove_from_index(&post.into()).await?;
 
         Ok(())
     }
 
+    /// Update a post and deliver the update
+    ///
+    /// # Panics
+    ///
+    /// This should never ever panic. If it does, create a bug report.
+    pub async fn update(&self, update_post: UpdatePost) -> Result<Post> {
+        if let Some(content) = update_post.content.as_ref() {
+            if content.chars().count() > self.instance_service.character_limit() {
```

Comment:
Can you just add a `// TODO(aumetra)` here mentioning a migration to `garde` for character limit enforcing?

I wanted to do it for a while now but always forgot

Reply:
You can mention #307 here